Makefile

```diff
@@ -11,6 +11,8 @@ REGRESS_OPTS = --inputdir=test
 PG_CPPFLAGS  = --std=c99
 include $(PGXS)
 
-ifeq ($(shell test $(VERSION_NUM) -lt 90600; echo $$?),0)
-REGRESS := $(filter-out parallel_test, $(REGRESS))
+ifeq ($(shell test $(VERSION_NUM) -lt 110000; echo $$?),0)
+# pg 11 added more parallel nodes which can break makes parallel test
+# for lower versions
+	REGRESS := $(filter-out 011_parallel_test, $(REGRESS))
```

Comment:
You can add additional expected file like on other extensions:
https://github.com/adjust/pg-base36/blob/master/test/expected/parallel_test_1.out